infrastructure/src/main/java/org/corfudb/infrastructure/datastore/DataStore.java

```diff
@@ -93,18 +97,15 @@ public DataStore(@Nonnull Map<String, Object> opts,
         return Caffeine.newBuilder().build(k -> null);
     }
 
-
-    public static int getChecksum(byte[] bytes) {
+    private static int getChecksum(byte[] bytes) {
```

Comment:
You can use `putBytes(ByteBuffer bytes)` directly

Reply:
See below comments.